### PR TITLE
Ethiopia launch

### PR DESCRIPTION
**Pull request by prabhanshuguptagit**
**Story card:** https://www.pivotaltracker.com/epic/show/4446092

## Because

We want to make some Ethiopia specific changes to the app. Deepa's notes: https://docs.google.com/document/d/1cDT6-MN5V1OUOWx-KPbZdNF-Z879KASJBCRhHS-YiE0/

## This addresses

- Translate the Patient and Facility Address model
- Placeholder locale files for Oromo and Tigrinya (to be filled in via transifex)
- Update the app manifests (except `production` and `demo`)
- Remove transgender option in gender
- Change SMS ISD code in country config
- Add a `ethiopia_medical_record` type to business identifiers

### CODE REVIEW

File: config/locales/address/en_ET.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+en_ET:
+  helpers:
+    label:
+      facility:
+        name: "Name"
+        facility_type: "Facility Type *"
+        size: "Size *"
+        street_address: "House and road number"
+        village_or_colony: "Kebele"
+        zone: "Zone"
```
Comment (prabhanshuguptagit) [inline config/locales/address/en_ET.yml:10]: Zone needs confirmation: Some regions have zonal administration some don't.

File: config/locales/address/en_ET.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+en_ET:
+  helpers:
+    label:
+      facility:
+        name: "Name"
+        facility_type: "Facility Type *"
+        size: "Size *"
+        street_address: "House and road number"
```
Comment (prabhanshuguptagit) [inline config/locales/address/en_ET.yml:8]: House and road number needs confirmation: A lot of districts don't have a house and road numbers
.

**Comment by prabhanshuguptagit** [comment]
> What test coverage is needed for these changes?

Added a spec for available genders. Don't think anything else needs a spec change.

**Comment by kitallis** [comment]
Looks good, let's merge this in so we can get started on translations ASAP.

**Pull request merged**